pyproject.toml

```diff
@@ -108,6 +108,10 @@ source = ["evap", "tools"]
 
 [tool.coverage.report]
 omit = ["*/migrations/*", "*__init__.py"]
+exclude_also = [
+    # mypy asserts these cases are not possible
+    "assert_never(.*)",
```

Comment:
Looks like a regex to me -- the parentheses then aren't literal parentheses, but it looks like that's what you wanted?